# Generated for NaoQi method ALLaser.laserON
# Enable laser light and sampling

---


